### ISSUE TITLE
Show progress-bar onsubmit form

### ISSUE THREAD

**Issue by AEK-BKF** [w3shaman/php-progress-bar#1]
Hi,
How can we use your code to show progress-bar on submit form using ajax ?

**Comment by w3shaman** [comment]
Yes. As you see in the sample code, I use "document ready" to trigger the Ajax call. You just need to change it so the Ajax call and also the progress checking process to run on form submission.

**Comment by kevingatp** [comment]
Hi, Is it possible to use in MVC framework such as Yii2?
Thanks

**Comment by w3shaman** [comment]
I never used Yii before but I think it is possible since the progress bar can be implemented in pure jQuery in PHP, so it can be implemented to in any PHP framework.

**Comment by kevingatp** [comment]
Thanks, you're right. It's work on Yii2 framework :) Big Thanks fot this awesome creation from cool creator 👍
